packages/mobile/src/components/ContextMenu/menus/InvitationContextMenu.container.tsx

```diff
@@ -82,7 +82,7 @@ export const InvitationContextMenu: FC = () => {
       title={'Add members'}
       items={items}
       hint={'Anyone with Quiet app can follow this link to join this community. Only share with people you trust.'}
-      link={invitationLink}
+      link={invitationLink.slice(0, 35) + '...'}
```

Comment:
Can't we do it with css text-overflow: ellipsis?

Reply:
This doesn't work in React Native from what I can see, but I found an equivalent to this - `numberOfLines={1}`